CHANGES.txt

```diff
@@ -1,3 +1,6 @@
+10.21.1 (July 27, 2022)
+- Added missed type definitions `enabled` from `sync`.
```

Comment:
what about `Bugfixing` entry instead of `Added`? to not confuse with a minor release